[2018-11-10 04:33:57,187] {models.py:1335} INFO - Dependencies all met for <TaskInstance: naveed_test.run_this_first_parallel 2018-11-10T12:22:44.367293+00:00 [queued]>
[2018-11-10 04:33:57,194] {models.py:1335} INFO - Dependencies all met for <TaskInstance: naveed_test.run_this_first_parallel 2018-11-10T12:22:44.367293+00:00 [queued]>
[2018-11-10 04:33:57,194] {models.py:1547} INFO - 
--------------------------------------------------------------------------------
Starting attempt 4 of 4
--------------------------------------------------------------------------------

[2018-11-10 04:33:57,223] {models.py:1569} INFO - Executing <Task(DummyOperator): run_this_first_parallel> on 2018-11-10T12:22:44.367293+00:00
[2018-11-10 04:33:57,223] {base_task_runner.py:124} INFO - Running: ['bash', '-c', u'airflow run naveed_test run_this_first_parallel 2018-11-10T12:22:44.367293+00:00 --job_id 93 --raw -sd DAGS_FOLDER/test.py --cfg_path /var/folders/zk/29b_jkbx6z58gzz4zv4tzd_40000gn/T/tmpY2v_5l']
[2018-11-10 04:34:00,030] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,029] {settings.py:174} INFO - setting.configure_orm(): Using pool settings. pool_size=5, pool_recycle=3600
[2018-11-10 04:34:00,051] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel /Users/naveed/anaconda2/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2018-11-10 04:34:00,051] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel   """)
[2018-11-10 04:34:00,202] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,201] {__init__.py:51} INFO - Using executor LocalExecutor
[2018-11-10 04:34:00,292] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel /Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/bin/cli.py:1595: DeprecationWarning: The celeryd_concurrency option in [celery] has been renamed to worker_concurrency - the old setting has been used, but please update your config.
[2018-11-10 04:34:00,293] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel   default=conf.get('celery', 'worker_concurrency')),
[2018-11-10 04:34:00,350] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,350] {cli.py:464} ERROR - Section ldap Option bind_user does not exist in the config!
[2018-11-10 04:34:00,351] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,350] {cli.py:464} ERROR - Section ldap Option search_scope does not exist in the config!
[2018-11-10 04:34:00,351] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,350] {cli.py:464} ERROR - Section ldap Option basedn does not exist in the config!
[2018-11-10 04:34:00,351] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option group_member_attr does not exist in the config!
[2018-11-10 04:34:00,352] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option uri does not exist in the config!
[2018-11-10 04:34:00,352] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option data_profiler_filter does not exist in the config!
[2018-11-10 04:34:00,352] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option bind_password does not exist in the config!
[2018-11-10 04:34:00,353] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option cacert does not exist in the config!
[2018-11-10 04:34:00,353] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option user_filter does not exist in the config!
[2018-11-10 04:34:00,353] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option user_name_attr does not exist in the config!
[2018-11-10 04:34:00,354] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,351] {cli.py:464} ERROR - Section ldap Option superuser_filter does not exist in the config!
[2018-11-10 04:34:00,354] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,352] {cli.py:464} ERROR - Section atlas Option username does not exist in the config!
[2018-11-10 04:34:00,355] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,352] {cli.py:464} ERROR - Section atlas Option host does not exist in the config!
[2018-11-10 04:34:00,355] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,352] {cli.py:464} ERROR - Section atlas Option password does not exist in the config!
[2018-11-10 04:34:00,355] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,352] {cli.py:464} ERROR - Section atlas Option port does not exist in the config!
[2018-11-10 04:34:00,355] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,352] {cli.py:464} ERROR - Section atlas Option sasl_enabled does not exist in the config!
[2018-11-10 04:34:00,356] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,353] {cli.py:464} ERROR - Section kubernetes Option logs_volume_subpath does not exist in the config!
[2018-11-10 04:34:00,356] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,353] {cli.py:464} ERROR - Section kubernetes Option git_repo does not exist in the config!
[2018-11-10 04:34:00,356] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,353] {cli.py:464} ERROR - Section kubernetes Option git_user does not exist in the config!
[2018-11-10 04:34:00,357] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,353] {cli.py:464} ERROR - Section kubernetes Option worker_container_repository does not exist in the config!
[2018-11-10 04:34:00,357] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,353] {cli.py:464} ERROR - Section kubernetes Option image_pull_secrets does not exist in the config!
[2018-11-10 04:34:00,357] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option git_sync_container_tag does not exist in the config!
[2018-11-10 04:34:00,357] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option in_cluster does not exist in the config!
[2018-11-10 04:34:00,357] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option namespace does not exist in the config!
[2018-11-10 04:34:00,358] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option dags_volume_subpath does not exist in the config!
[2018-11-10 04:34:00,358] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option git_sync_init_container_name does not exist in the config!
[2018-11-10 04:34:00,359] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option dags_volume_claim does not exist in the config!
[2018-11-10 04:34:00,359] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option git_subpath does not exist in the config!
[2018-11-10 04:34:00,359] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option worker_service_account_name does not exist in the config!
[2018-11-10 04:34:00,359] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option git_branch does not exist in the config!
[2018-11-10 04:34:00,360] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option logs_volume_claim does not exist in the config!
[2018-11-10 04:34:00,360] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,354] {cli.py:464} ERROR - Section kubernetes Option gcp_service_account_keys does not exist in the config!
[2018-11-10 04:34:00,360] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section kubernetes Option airflow_configmap does not exist in the config!
[2018-11-10 04:34:00,361] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section kubernetes Option git_sync_container_repository does not exist in the config!
[2018-11-10 04:34:00,361] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section kubernetes Option worker_container_tag does not exist in the config!
[2018-11-10 04:34:00,361] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section kubernetes Option git_password does not exist in the config!
[2018-11-10 04:34:00,362] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section kubernetes Option delete_worker_pods does not exist in the config!
[2018-11-10 04:34:00,362] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section lineage Option backend does not exist in the config!
[2018-11-10 04:34:00,362] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section hive Option default_hive_mapred_queue does not exist in the config!
[2018-11-10 04:34:00,363] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,355] {cli.py:464} ERROR - Section dask Option cluster_address does not exist in the config!
[2018-11-10 04:34:00,363] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,356] {cli.py:464} ERROR - Section dask Option tls_cert does not exist in the config!
[2018-11-10 04:34:00,363] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,356] {cli.py:464} ERROR - Section dask Option tls_key does not exist in the config!
[2018-11-10 04:34:00,363] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,356] {cli.py:464} ERROR - Section dask Option tls_ca does not exist in the config!
[2018-11-10 04:34:00,363] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,356] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_host does not exist in the config!
[2018-11-10 04:34:00,364] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,356] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_log_id_template does not exist in the config!
[2018-11-10 04:34:00,364] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,356] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_end_of_log_mark does not exist in the config!
[2018-11-10 04:34:00,364] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,361] {models.py:258} INFO - Filling up the DagBag from /Users/naveed/Mira/hackathon/airflow/dags/test.py
[2018-11-10 04:34:00,430] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,430] {example_kubernetes_operator.py:54} WARNING - Could not import KubernetesPodOperator: No module named kubernetes
[2018-11-10 04:34:00,431] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,430] {example_kubernetes_operator.py:55} WARNING - Install kubernetes dependencies with:     pip install airflow['kubernetes']
[2018-11-10 04:34:00,569] {base_task_runner.py:107} INFO - Job 93: Subtask run_this_first_parallel [2018-11-10 04:34:00,569] {cli.py:492} INFO - Running <TaskInstance: naveed_test.run_this_first_parallel 2018-11-10T12:22:44.367293+00:00 [running]> on host naveeds-MacBook-Pro.local
[2018-11-10 04:34:02,155] {logging_mixin.py:95} INFO - [2018-11-10 04:34:02,154] {jobs.py:2612} INFO - Task exited with return code 0

